torchinductor/ir.py

```diff
@@ -2840,6 +2840,49 @@ def __init__(self, layout, input, index):
     def should_allocate(self):
         return False
 
+class LinearReLU(ExternKernelAlloc):
```

Comment:
`LinearReLU` is specific for `Linear` + `ReLU`. Should this class be `LinearEltwise`?

Reply:
Fixed the name.